scripts/simulator/simulate.py

```
@@ -0,0 +1,467 @@
+#!/usr/bin/env python3
```

Ruff (0.13.1)

Shebang is present but file is not executable

(EXE001)